code/game/gamemodes/crew_objective/crew_objective.dm

```diff
@@ -362,8 +362,7 @@
 */
 /datum/crew_objective/proc/fade()
 	status	=	CO_STATUS_FADING
-	if (is_processing)
```

Comment:
why remove this check?

Reply:
Because it checks if it is processing in STOP_PROCESSING()